src/migrations/create.rs

```diff
@@ -945,3 +949,21 @@ fn add_newline() {
     assert_eq!(wrapper("(1 + 7) #xx"), "(1 + 7) #xx\n");
     assert_eq!(wrapper("(1 #one\n + 3 #three\n)"), "(1 #one\n + 3 #three\n)");
 }
+
+#[tokio::test]
+ async fn start_migration() {
+    let schema_dir = PathBuf::from("/Users/dijanapavlovic/Documents/projects/edgedb/edgedb-cli/tests/migrations/db5");
```

Comment:
This won't work on anyone else's machine (and is probably why the tests fail)

Reply:
ahaha omg yes :D will fix it :D